components/organisms/community/Comment.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+import { COLOR } from '@/styles/color';
+import { Icon } from '@/components/atoms';
+
+type CommentProps = {
+  optionHandler: (type: string) => void;
+  menuOpenHandler: () => void;
+};
+
+function Comment({ optionHandler, menuOpenHandler }: CommentProps) {
+  return (
+    <Wrapper>
+      <HeaderBlock>
+        <Username>익명 친구</Username>·<RegTime>115:31</RegTime>
+        <Icon.More
+          onClick={() => {
+            menuOpenHandler();
+            optionHandler('comment');
+          }}
+          style={{ position: 'absolute', right: 0, top: '50%', transform: 'translate(-50%, -50%)' }}
+        />
+      </HeaderBlock>
+
+      <ContentBlock>나도 담주에 노티드 갈건데 하는데 달달한거 먹고 기분 풀렸다니 다행이야. 힘내자</ContentBlock>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.article`
+  padding: 9px 15px;
+  border-bottom: 1px solid #f2f2f2;
+`;
+
+const HeaderBlock = styled.div`
+  display: flex;
+  position: relative;
+`;
+
+const ContentBlock = styled.div`
+  padding-top: 4px;
+  padding-bottom: 15px;
+  font-size: 15px;
+  color: ${COLOR.GRAY07};
+`;
```

Comment:
`{COLOR}` 변수를 가져오는 대신 `theme` Props 를 통해 표현 할 수 있습니당

```suggestion
const ContentBlock = styled.div`
  padding-top: 4px;
  padding-bottom: 15px;
  font-size: 15px;
  color: ${({ theme }) => ${theme.GRAY07}};
`;
```

Reply:
네 수정할게요~